src/grt/src/RepairAntennas.h

```diff
@@ -214,6 +214,7 @@ class RepairAntennas
   void destroyNetWires(const std::vector<odb::dbNet*>& nets_to_repair);
   odb::dbMTerm* findDiodeMTerm();
   double diffArea(odb::dbMTerm* mterm);
+  bool runJumperInsertion() { return run_jumper_insertion_; }
```

Comment:
This function name can mislead to what it actually does. Maybe instead of run... you can use `isJumperInsertion`.